### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA are not announcing the name of the Destination Folder edit in "Create Authentication Files dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1458]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , tab to Destination Folder edit and observe the behaviour of Jaws and NVDA

**Actual Result:**
Jaws and NVDA are not announcing the name of the edit box

**Jaws behaviour:**
read-only edit ,
 C:\Users\abby, 

**NVDA behaviour:**
 edit ,
 C:\Users\abby, 

**Expected Result:**
Jaws and NVDA should announce the name of the edit box i.e Destination Folder

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![create authntication files-edit](https://user-images.githubusercontent.com/22956130/36099586-45f2d29a-1029-11e8-8e44-d423c820f6ad.jpg)
![ca0e05d7](https://user-images.githubusercontent.com/22956130/36099588-49fdfb9e-1029-11e8-822c-177d2eee18d7.jpg)
![d183e6ac](https://user-images.githubusercontent.com/22956130/36099594-4dace746-1029-11e8-80a5-f19124a8d1ee.jpg)

**Comment by Eskibear** [comment]
According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community. 
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by Eskibear** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all. 

* **Importance of experience**: it depends, varies from `somewhat important` to `important`.
* **Quality of experience**: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by soumyatippapuram** [comment]
This issue is still reproducing in latest test pass 
(IntelliJ IDEA 2019.1.3   Build #IU-191.7479.19)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749934
